WordPress/Classes/ViewRelated/Reader/ReaderTagCardCellViewModel.swift

```diff
@@ -177,6 +163,49 @@ private extension ReaderTagCardCellViewModel {
 
         return snapshot
     }
+
+    private func createDataSource(with collectionView: UICollectionView) -> DataSource {
+        let dataSource = DataSource(collectionView: collectionView) { [weak self] collectionView, indexPath, item in
+            switch item {
+            case .empty:
```

Comment:
(Nit, cleanup) Now that the cell provider implementation is inline, we should be able to delete the (now) unused method `cardCellProvider` [at line 120](https://github.com/wordpress-mobile/WordPress-iOS/pull/23183/files#diff-4b438f52519c6a875387d3a14901e3ec4f1561db7a952962c893723808d428fcR120).

Reply:
Good call, I forgot to do that. I removed it in https://github.com/wordpress-mobile/WordPress-iOS/pull/23183/commits/6f3f7d54e317e5d47d8ab010d9481ee9b9460319.